nav2_core/package.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>nav2_core</name>
+  <version>0.2.2</version>
+  <description>A set of headers for plugins core to the navigation2 stack</description>
+  <maintainer email="stevenmacenski@gmail.com">Steve Macenski</maintainer>
+  <license>BSD-3-Clause</license>
```

Comment:
You prefer BSD-3-Clause over Apache 2.0 here? We prefer Apache, as does the upstream ROS2 (like rclcpp), but have a mixture of both in the repo. I just want to make sure before merging that's what you/we want.

Reply:
So yeah, I expected a conversation to get started on this one.

Given how core this is to everything up stream of it, the thought was that since this is foundational for things above it, I didn't want to stack an Apache license on a BSD implemented plugin, that makes things real weird. I'm not a lawyer and since BSD is the "freest of them all" I thought it's good to have it start there.

In particular since navfn & DWB are BSD, I didn't want to force them by using the headers to have issues with licenses. If someone said "I talked to / am a lawyer, that's fine" then I'd move to Apache without issue

Reply:
@crdelsey - do you have a preference here, you just resolved the other license issues we had. I'm actually fine with BSD here but want to check

Reply:
I'm also more than happy to start off with BSD and if we find out later we can move the Apache without causing issues for BSD plugins, I approve those changes over. Samsung prefers me to use Apache 2.0 as well, the only reason this is BSD is because of the BSD things that sit on top of it (and also largely inspired from other BSD packages)

Reply:
I did some reading, Apache should be fine

Reply:
updating.